packages/agent-app-sdk/README.md

```diff
@@ -247,6 +247,14 @@ This widget currently does not support any events.
 
 Displays a red badge on top of the Fullscreen app icon. Use this to notify Agents there’s something important inside the widget. Make sure Agents can dismiss the notification to avoid cluttered UI.
 
+#### `navigate(pathname: string): Promise<void>`
+
+Navigates LiveChat Agent App to given pathname.
+
+#### `setReportsFilters(filters: any): Promise<void>`
```

Comment:
Could we use dedicated types? `any` seems very "unhelpful"

Reply:
Fixed in [5e45d12](https://github.com/livechat/products-sdk/pull/26/commits/5e45d128e28719ff10b82722c6d8f978430d9146)